pkg/util/replicaset_label_test.go

```diff
@@ -0,0 +1,61 @@
+// Copyright (c) 2024 VMware, Inc. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package util_test
+
+import (
+	"encoding/base64"
+	"fmt"
+	"hash/fnv"
+	"strings"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+
+	"github.com/vmware-tanzu/vm-operator/pkg/util"
+)
+
+var _ = Describe("FormatValue", func() {
+	Context("input is a valid label value", func() {
+		It("returns the input label", func() {
+			inputLabel := "valid-k8s-label-value"
+			ret, err := util.FormatValue(inputLabel)
+			Expect(err).ToNot(HaveOccurred())
+			Expect(ret).To(Equal(inputLabel))
+		})
+	})
+
+	Context("input is not a valid label", func() {
```

Comment:
nit: To keep wording consistent with below. `input is invalid label`, `does not panic, returns a hashed label value`